packages/squiggle-lang/src/rescript/Distributions/DistributionOperation.res

```diff
@@ -263,6 +267,8 @@ module Constructors = {
   let normalize = (~env, dist) => C.normalize(dist)->run(~env)->toDistR
   let isNormalized = (~env, dist) => C.isNormalized(dist)->run(~env)->toBoolR
   let klDivergence = (~env, dist1, dist2) => C.klDivergence(dist1, dist2)->run(~env)->toFloatR
+  let logScoreWithPointResolution = (~env, prediction, answer, prior) =>
```

Comment:
Given this has 4 paramaters, it would be good to name them. This stuff gets really confusing imo.